index.html

```diff
@@ -9,11 +9,20 @@
   </head>
   <body>
     <header>
-      <h1>Awesome Books</h1>
+      <h1>Awesome books</h1>
+      <nav class="nav-bar">
+        <ul class="nav-list">
+          <li class="list">List</li>
+          <li class="add-new">Add New</li>
+          <li class="contact">Contact</li>
+        </ul>
+      </nav>
     </header>
-    <section class="books"></section>
-    <hr />
-    <section class="contact-section">
+    <section class="books">
+      <h2>All Awesome Books</h2>
+      <article class="book-list"></article>
+    </section>
+    <section class="add-section">
       <h2 class="add-header">Add a new book</h2>
       <form
```

Comment:
- [ ] Kindly clean the input fields when a user adds a new book so that a user should be aware that he or she has created a book already.

![image](https://user-images.githubusercontent.com/55635977/165523934-cbb2ad9a-7b65-47d4-afe2-f898e0268042.png)

Reply:
@Emmyn5600
it makes sense but, does that exists in the requirements?